day2/02_Integrations/02_IDEs.md

```diff
@@ -1,10 +1,11 @@
 !SLIDE smbullets
 # IDEs
 
-* Atom
 * JetBrains PHPStorm, ...
 * Visual Studio
+* VSCodium
```

Comment:
I think we should use the full name Visual Studio Code - or am I thinking of something else right now?

Reply:
afaik VSCodium is the OSS variant (MS-telemetry free).

Reply:
Yeah I think you're correct

Reply:
I will fix it on this branch

Reply:
Done :heavy_check_mark: